magenta/video/tools/random_pick.py

```diff
@@ -20,10 +20,10 @@
 
 import argparse
 import glob
-import ntpath
-import os
 import random
 import shutil
+import os
```

Comment:
why are we breaking alphabetical ordering in these places?

Reply:
@adarob 
Manually changed orders due to **pylint** which throwing:
 `C0411: 24 : standard import "import os" should be placed before "import ntpath" [wrong-import-order]`
Full details the issue, link https://github.com/tensorflow/magenta/issues/1558

Imports were fixed automatically in the new commit https://github.com/tensorflow/magenta/pull/1559/commits/752621ddf5bb48b599210c5e1a1da62f342ed5cc by running `isort -rc .`, link https://github.com/timothycrosley/isort#using-isort